FeedStoreChallenge/CoreDataFeedStore.swift

```diff
@@ -29,14 +29,52 @@ public final class CoreDataFeedStore: FeedStore {
 	}
 
 	public func retrieve(completion: @escaping RetrievalCompletion) {
-		fatalError("Must be implemented")
+		perform { context in
+			do {
+				let fetchedCache = try ManagedCache.find(in: context)
+				if let cache = fetchedCache {
```

Comment:
`fetchedCache` is not used anywhere except the `if let ...` block. The less variables the less developer should remember while reading the code. Thus making it a one-liner will improve readability.

```suggestion
				if let cache = try ManagedCache.find(in: context) {
```